### PR TITLE
Assignment 4 @grahamna @BenKnoll3 New Submission

### PR DESCRIPTION
**Pull request by grahamna**
the database parts aren't really ready yet, but much of the skelleton of what we need is here.

@grahamna
@BenKnoll3

Site: https://green-sea-09672851e.3.azurestaticapps.net/

db wordleweb2023/Assignment4 : https://wordleweb2023.database.windows.net/

I've set up a second database called Assignment4, and changed the DefaultConnection string to DefaultConnection2, pointing at the new database, but it doesn't seem like it's taking yet... I'll keep working on it.

The first pull request was attempting to merge their Assignment 4 with their main...

EDIT: Database is now hooked up and working with the rest of the project. Backgrounds were aquired. Please note, that due to local caching and stuff you may need to open the site on incognito mode to create "new" users, or to be "assigned" a fresh Guid for your session. If you "look" into the database and you aren't there, some functions kinda break.

NOTE: Meg, or mysterious grader, BenKnoll3 participated more into this assignment than what it seems, there was one or twice that i merged his branch into my branch before pushing to Assignment 4 to make edits and fixes.

### CODE REVIEW

**Comment by tienv9** [comment]
Make all pages responsive ✅
1080p desktop ✅
iPad Air ✅
Samsung Galaxy S20 Ultra Super Mega Excellent ✅
IPhone SE ✅

Create a improved landing page and site theme (Background, make it look attractive to play) ❌
Theme should extend all pages ❌
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ❌
Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ✅
It should indicate if the user has played that day ❌
Clicking on the item should take the player to that game (Extra Credit) ❌

Other than the db needs to be connected to work probably, and the 'how to play' being way too complicated and wordy. You guys seem to be on track with the initial setup and doing a good job.

File: wordle-web/src/views/DailyWordListView.vue [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      class="align-center text-center mx-auto"
+      max-height="100%"
+      max-width="90%"
+      min-height="90%"
+      min-width="25%"
+      rounded
+    >
+      <v-card-title class="titleText">Word Of The Day Games! (Dummy Data)</v-card-title>
+      <v-card-text>
+        <v-row justify="center">
```
Comment (mmoran314) [inline wordle-web/src/views/DailyWordListView.vue:13]: I really like how these are all buttons that take you to the word for that day. Especially with the color. I think I may just rearrange this to be from top to bottom when listing all of them to make it clear the order of when the word was the word of the day.

**Review by mmoran314 — COMMENTED** [review]
Make all pages responsive ✅
1080p desktop ✅
iPad Air ✅
Samsung Galaxy S20 Ultra Super Mega Excellent ✅
IPhone SE ✅❌ logo gets cut off here but not too big of a deal

Create a improved landing page and site theme (Background, make it look attractive to play) ❌
Theme should extend all pages ✅
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ❌
Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ✅
It should indicate if the user has played that day ❌✅ I'm assuming the color of the button indicated this.
Clicking on the item should take the player to that game (Extra Credit) ❌✅

It looks like this is going in the right direction in a lot of areas. I really like some of your implementations, especially for the previous words. Just needs the tough parts of getting them from the database.

File: wordle-web/src/views/DailyWordListView.vue (hunk 1 above)
Comment (grahamna) [reply wordle-web/src/views/DailyWordListView.vue:13]: Yeah, the data should be fetched ordered from the axios web api call. Left to right, top to bottom. I'll make sure this is the case once the web database is populated and we can fetch stuff.

**Comment by grahamna** [comment]
> Make all pages responsive ✅ 1080p desktop ✅ iPad Air ✅ Samsung Galaxy S20 Ultra Super Mega Excellent ✅ IPhone SE ✅
> 
> Create a improved landing page and site theme (Background, make it look attractive to play) ❌ Theme should extend all pages ❌ Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅ Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ❌ Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ✅ It should indicate if the user has played that day ❌ Clicking on the item should take the player to that game (Extra Credit) ❌
> 
> Other than the db needs to be connected to work probably, and the 'how to play' being way too complicated and wordy. You guys seem to be on track with the initial setup and doing a good job.

I just copy pasted someone else's instructions. Clicking on the red/green buttons in the daily words does take you to the individual games and they are red/green... I'll get on making a theme I guess?

**Review by mmwoodfo — COMMENTED** [review]
Looks like you might have a bug with your word of the day & linking to a specific word for a day.

If I play the word of the day... then navigate to today's specific day to play that word, it turns out to be a different word from the word of the day, even though it's the word for today.

**Pull request closed without merging**